### ISSUE TITLE
Traefik Rule with Path

### ISSUE THREAD

**Issue by dirosden** [rawmind0/rancher-traefik#15]
hi, have got a traefik instance working routing calls to backend services. but i have a scenario where the backend services routing rule is based on the path, rather than the domain. currently my docker inspect has these:

```
            "traefik.enable": "true",
            "traefik.frontend.entryPoints": "http",
            "traefik.frontend.passHostHeader": "true",
            "traefik.frontend.rule": "Host:apigwy.productgwy.localhost;Path:/product",
            "traefik.path": "/product",
            "traefik.port": "8081",
```

But the rule that gets registered within traefik is:
       Route    Rule
       service  Host:apigwy.product

The scenario I am trying to achieve is as below:
- Each Stack gets a dedicated URL path on a common domain; For example, common domain will be api.domain.com;
- Each of the Stacks handle calls to a particular path; for example, /product, /service etc.
- Each stack will set the appropriate Labels similar to the Domain and Port

Is this possible to be configured? 

My current docker-compose is like this:

apigwy:
  image: <<image>>
  labels:
    - "traefik.enable=true"
    - "traefik.port=8081"
    - traefik.frontend.rule=Path:${listenat}

The Rancher Compose captures the path and sets it.

Thanks

**Comment by rawmind0** [comment]
Hi @dirosden 

thanks for use traefik with rancher and the feedback.

I'm sorry, but by the moment, path filter label is not implemented. The same issue is here.....
https://github.com/rawmind0/alpine-traefik/issues/16

I'm working on that. Shortly, it would be implemented. :)

Best regards....

**Comment by rawmind0** [comment]
Hi @dirosden ...

added the feature of traefik.path label in the last rancher-traefik version.

You could try it adding https://github.com/rawmind0/service-catalog catalog to your rancher server...

Best regards...

**Comment by macneib** [comment]
Hi @rawmind0, path seems to be working as you intended.  Nice work, and thank you!
Quick question about one last step I'm curious about.

Let's say I've got a container/serivce running in a `www` stack in an environment called `dev` .
assuming I set `traefik.domain : dev.example.com and`traefik.path : /` then I should expect  traefik to configure the service as follows:

`Host:www.dev.oak.cloud;Path:/`

What would it take to just get the host down to only `dev.example.com` and so I can land my api services (sitting in another stack called `api` but same dev environment) so the would be something like 

`dev.example.com/api/v1/foo` or in traefik terms: `Host:dev.example.com; Path:/api/v1/foo` 

instead of `Host:api.dev.example.com;Path:/api/v1/foo`

or other services i.e. 

`dev.example.com/api/v1/bar` `dev.example.com/auth/local` 

so I would like to know if it is possible to set the same host throughout the stacks, but only distinguish services though the `Path` option. 

Basically I'm unsure how to set the host as static `dev.example.com` so I can attach services only through Path.

Best regards,
macneib 

edit:   I just just check some of the other issue discussions, and I apparently am just asking for the same thing.  Please disregard. I'll wait for a solution.

**Comment by rawmind0** [comment]
Hi @macneib ...

yes, the question is a little bit complicated by how this package was thought, as a system service. If you already read some other issues, you may know. 

I'm already working to be able to select a different traefik scopes when you deploy, then you could deploy one traefik pool per stack, instead a global one, and you could repeat names, if you need... 

A little bit of patience.... :P

Best regards....

**Comment by macneib** [comment]
Sounds good, no rush.

Best regards!

**Comment by rawmind0** [comment]
Hi @macneib  ...

Look at last commits... Added traefik.alias label to create host rules $traefik.alias.$traefik.domain ..
It creates host rules $service_name.$stack_name.$traefik.domain too...

May it work for you??

**Comment by macneib** [comment]
Hi @rawmind0 

This works for me.  Much appreciated!

Thanks again and best regards!
